### ISSUE TITLE
Deprecated

### ISSUE THREAD

**Issue by lordofscripts** [thatisuday/go-cross-build#12]
Tried to integrate this workflow but it fails. The error given in the GitHub Actions tab says:

"```
The following actions uses node12 and will be forced to run on node16:actions/checkout@v2
```."

**Comment by chenasraf** [comment]
In the workflow yaml, update `actions/checkout@v2` to `actions/checkout@v4`

**Comment by lordofscripts** [comment]
Yes, I did that and setup to V5 as well. But I forgot to update this because I was getting too many issues with this thisisuday and had to switch to a different workflow that doesn't use this.

In short, I dropped this `go-cross-build` in favor of another workflow which was less problematic. Had to make a few tweaks but finally worked.

**Comment by AhmedAbouelkher** [comment]
> Yes, I did that and setup to V5 as well. But I forgot to update this because I was getting too many issues with this thisisuday and had to switch to a different workflow that doesn't use this.
> 
> In short, I dropped this `go-cross-build` in favor of another workflow which was less problematic. Had to make a few tweaks but finally worked.

Would you list the alternative workflow you used?

**Comment by lordofscripts** [comment]
> 
> Would you list the alternative workflow you used?

Check my current workflow at [https://GitHub.com/lordofscripts/wipechromium/](https://GitHub.com/lordofscripts/wipechromium/)

It's not perfect but it is building for several OS & platforms and adding it to a draft release. What it doesn't do us:

- compress the deliverable in an OS-specific format (zip/tar.gz)
- use `.exe` extension for Windows
